apps/asap-server/test/controllers/calendars.test.ts

```diff
@@ -153,14 +153,22 @@ describe('Calendars controller', () => {
         });
       });
 
-      test('Should skip the calendars which do not belong to any group', async () => {
+      test('Should show the calendars which do not belong to any group', async () => {
         const squidexGraphqlResponse = getSquidexCalendarsGraphqlResponse();
 
         const calendar1 = getSquidexGraphqlCalendar();
+        calendar1.flatData.googleCalendarId = 'calendar1@google.com';
         calendar1.referencingGroupsContents = null;
         const calendar2 = getSquidexGraphqlCalendar();
+        calendar2.flatData.googleCalendarId = 'calendar1@google.com';
```

Comment:
i think if you update the googleCalendarId to the same value for each of the calendars then it doesnt really make much sense anyway does it?

Reply:
because there is maybe in the type - if you do not assign the googleCalendarId it shows that there could be an issue :P